CMakeLists.txt

```
@@ -399,7 +399,9 @@ add_subdirectory (io)
 add_subdirectory (net)
 add_subdirectory (graf2d)
 add_subdirectory (graf3d)
-add_subdirectory (gui)
+if(gui)
```

I would like to suggest to moving to
```suggestion
if(NOT minimal)
```
Because you either build minimal, or gminimal, or "standard" ROOT. A fourth option doesn't seem to work.